crates/iroha_core/src/kura.rs

```diff
@@ -413,10 +434,10 @@ impl BlockStore {
             .read(true)
             .open(path.clone())
             .add_err_context(&path)?;
-        let start_location = start_block_height * (2 * std::mem::size_of::<u64>() as u64);
+        let start_location = start_block_height * BlockIndex::SIZE;
```

Comment:
Should we handle potential overflow?

Reply:
That would happens somewhere around $18,446,744,073,709,551,615 / 16$ block height, which isn't very realistic to handle, IMO.